Library/Homebrew/cmd/update-report.rb

```diff
@@ -158,6 +159,30 @@ def migrate_legacy_cache_if_necessary
       end
     end
   end
+
+  def link_manpages
+    return if HOMEBREW_PREFIX.to_s == HOMEBREW_REPOSITORY.to_s
+    conflicts = []
+    Dir["#{HOMEBREW_REPOSITORY}/share/man/man1/*.1"].each do |src|
+      src = Pathname(src)
```

Comment:
Consistency with the rest of Homebrew (this is a few days old, before Cask was merged):

```console
$ ack 'Pathname\(' | wc -l
       9
$ ack 'Pathname\.new\(' | wc -l
     128
```